libs/utils/utils.js

```diff
@@ -764,8 +764,9 @@ async function decorateIcons(area, config) {
   const { base } = config;
   loadStyle(`${base}/features/icons/icons.css`);
   loadLink(`${base}/img/icons/icons.svg`, { rel: 'preload', as: 'fetch', crossorigin: 'anonymous' });
```

Comment:
Does this preload need to change, do we always load from a federal icon sheet now? If we know the icons are _always_ coming from there, we should change this

Reply:
Since these are all individual icons now, I think we actually should remove this line since it's obsolete. We could pre-load all svg icons, but would have to do some type of loop or query to target them all. LKM if you have any thoughts. Thanks

Reply:
Unless we have an _easy_ way to know which icons to load it's probably best to just remove